.github/workflows/tomcat-https-multi-certificate-test.yml

```diff
@@ -0,0 +1,177 @@
+          docker rm -f pki-dist
+
+      - name: Install packages
+        run: |
+          docker cp /tmp/RPMS/. server:/root/RPMS/
+          docker exec server bash -c "dnf install -y /root/RPMS/*"
+
+      - name: Create Tomcat
+        run: |
+          docker exec server pki-server create -v
+
+      - name: Create NSS database in Tomcat
+        run: |
+          docker exec server pki-server nss-create --no-password
+
+      - name: Create RSA SSL server cert
+        run: |
+          docker exec server pki \
+              -d /var/lib/pki/pki-tomcat/alias \
+              nss-cert-request \
+              --subject "CN=server.example.com" \
+              --ext /usr/share/pki/server/certs/sslserver.conf \
+              --csr sslserver-RSA.csr
+
+          docker exec server pki \
+              -d /var/lib/pki/pki-tomcat/alias \
+              nss-cert-issue \
+              --csr sslserver-RSA.csr \
+              --ext /usr/share/pki/server/certs/sslserver.conf \
+              --cert sslserver-RSA.crt
+
+          docker exec server pki \
+              -d /var/lib/pki/pki-tomcat/alias \
+              nss-cert-import \
+              --cert sslserver-RSA.crt \
+              sslserver-RSA
+
+      - name: Create EC SSL server cert
+        run: |
+          docker exec server pki \
+              -d /var/lib/pki/pki-tomcat/alias \
+              nss-cert-request \
+              --key-type EC \
+              --subject "CN=server.example.com" \
+              --ext /usr/share/pki/server/certs/sslserver.conf \
+              --csr sslserver-EC.csr
+
+          docker exec server pki \
+              -d /var/lib/pki/pki-tomcat/alias \
+              nss-cert-issue \
+              --csr sslserver-EC.csr \
+              --ext /usr/share/pki/server/certs/sslserver.conf \
+              --cert sslserver-EC.crt
+
+          docker exec server pki \
+              -d /var/lib/pki/pki-tomcat/alias \
+              nss-cert-import \
+              --cert sslserver-EC.crt \
+              sslserver-EC
+
+      - name: Create HTTPS connector
+        run: |
+          docker exec server pki-server jss-enable
+
+          docker exec server pki-server http-connector-add \
+              --port 8443 \
+              --scheme https \
+              --secure true \
+              --sslEnabled true \
+              --sslProtocol SSL \
+              --sslImpl org.dogtagpki.jss.tomcat.JSSImplementation \
+              Secure
+
+      - name: Add HTTPS connector certificates
+        run: |
+          docker exec server pki-server http-connector-cert-add \
+              --keyAlias sslserver-EC \
+              --keystoreType pkcs11 \
+              --keystoreProvider Mozilla-JSS \
+              EC
+
+          docker exec server pki-server http-connector-cert-add \
+              --keyAlias sslserver-RSA \
+              --keystoreType pkcs11 \
+              --keystoreProvider Mozilla-JSS \
+              RSA
+
```

Comment:
I'm curious, provided that the client supports them, how NSS does pick between 2 RSA certs. Is it by key size? order of config? What about 2 EC curves?  Anyway, if it's determined that it's by the order of configuration, at least we know what to tell admins in the documentation.
Speaking of documentation, it doesn't have to be right now, but when ML-DSA is officially introduced for cs, we want to add config info to doc.
For now, it's up to you whether you want to add tests for 2 RSA certs with different sizes, or 2 EC certs with different curves.